src/components/js/EcAppCard.js

```diff
@@ -11,7 +14,16 @@ export default {
       type: Boolean
     }
   },
+  methods: {
+    i18n
```

Comment:
Você não usa o método mais, certo? Só usa o função que você importou, não precisa do `this.i18n`...

```suggestion
```